### ISSUE TITLE
Deleted tooltip property from SubmitAction adaptivecard component on version 4.23.0

### ISSUE THREAD

**Issue by luisfelipe998** [microsoft/botbuilder-js#4735]
## Versions
What package version of the SDK are you using? 4.23.0
What nodejs version are you using? v20.17.0
What os are you using? MacOS 14.6

## Describe the bug
On version 4.22.3, SubmitAction button has a `tooltip` property which was deleted on the update to version 4.23.0. This change broke my code.

## To Reproduce
1. Install version 4.22.3
2. Create a submitAction button like this and see it works:
```
import { SubmitAction } from "adaptivecards";
const action = new SubmitAction();
action.tooltip = "my tooltip";
```
3. Update to version 4.23.0
4. See that `tooltip` property is not valid anymore.

## Expected behavior
I expect that SubmitAction remains with tooltip and all its other properties on minor updates of the dependency.

**Comment by tracyboehrer** [comment]
@luisfelipe998 We'll take a look.  The adaptivecards dependency was added in Nov 2023 (version 1.2.3).  But we didn't change our direct dependency for that in 4.23.0.  What version is it resolving to for you?

**Comment by luisfelipe998** [comment]
Hi @tracyboehrer, this is the resolved version on yarn.lock:

```
adaptivecards@1.2.3:
  version "1.2.3"
  resolved "https://registry.yarnpkg.com/adaptivecards/-/adaptivecards-1.2.3.tgz#b69319c75bd214a5ec48119240c98449e7df685a"
  integrity sha512-amQ5OSW3OpIkrxVKLjxVBPk/T49yuOtnqs1z5ZPfZr0+OpTovzmiHbyoAGDIsu5SNYHwOZFp/3LGOnRaALFa/g==

adaptivecards@~2.10.0:
  version "2.10.0"
  resolved "https://registry.yarnpkg.com/adaptivecards/-/adaptivecards-2.10.0.tgz#1c94e84491afe5a4f2d4060f6e0fc57b895205e3"
  integrity sha512-QcgXGFlX3rgIfZkxvQY896zEVBh30mE37z3XKhcQW2OqSUrVH6dNT78MkM//W4wRdrZizSHqXKLaKyVVxAgVFg==
```

**Comment by luisfelipe998** [comment]
I saw that I have this dependency `"@microsoft/adaptivecards-tools": "^1.3.2"` that was resolving `adaptivecards@~2.10.0`.
On version 4.22.3 of botbuilder, The package.json of the adaptivecards was at 2.10.0 inside node_modules. When I updated to 4.23.0, it changed to version 1.2.3. Nevertheless, I'm closing this issue because botbuilder kept using version 1.2.3, so this is as issue on my side.

Thanks for your assistance.
